terraform/fullnode/vultr/cluster.tf

```diff
@@ -0,0 +1,16 @@
+resource "vultr_kubernetes" "k8" {
+  region  = var.fullnode_region
+  label   = "aptos-devnet"
```

Comment:
Assume this is the k8s cluster name on vultr? could you make this the same name as the terraform workspace? so that people can create multiple clusters if they want and the name won't be conflicting with each other. you can do something like `label = "aptos-${terraform.workspace}"`

Reply:
Done.